tests/regression/tests/REQUEST-934-APPLICATION-ATTACK-GENERIC/934110.yaml

```diff
@@ -0,0 +1,42 @@
+---
+  meta:
+    author: "Andrea Menin (theMiddle)"
+    description: "XML eXternal Entity"
+    enabled: true
+    name: 934110.yaml
+  tests:
+  -
+    test_title: 934110-1
+    desc: "Test XXE in file upload"
+    stages:
+    - stage:
+        input:
+          dest_addr: "127.0.0.1"
+          headers:
+            Host: "localhost"
+            User-Agent: "ModSecurity CRS 3 Tests"
+            Content-Type: "multipart/form-data; boundary=----WebKitFormBoundarysbZ9sZS2ONRQAD7q"
```

Comment:
Be aware that in the test the boundary starts with 4 dashes, but then below there are 6 dashes, so it won't work properly....

Reply:
uhmm, to be honest, I copied the raw request generated by my browser (let me check it again). But if I'm not wrong the two additional hyphens should be placed at each part and not in the boundary name on CT.

I need to check the RFC but it should be something like
```
Content-Type: multipart/form-data; boundary=[name]

--[name]
...
```

Reply:
anyway, I tried to add two more hyphens in the boundary name but nothing changed.

Reply:
I misread it, and it was my bad. It was right the first time. Sorry.